packages/components/src/interfaces/complex/Hero.ts

```diff
@@ -26,7 +26,7 @@ const HeroButtonsSchema = Type.Object({
     Type.String({
       title: "Button destination",
       description: "When this is clicked, open:",
-      format: "link",
+      format: "uri",
```

Comment:
what's the difference between `uri` and `link`?

Reply:
Hmm no difference, mainly a naming convention. `uri` is one of the formats in the JSON Schema specs, but since we are supporting references as links, it might be more accurate to call it `link` to differentiate. Updated in 0c0bd6f16ce7b993d3249f5a0d70c681819a65ac.